.dockerignore

```diff
@@ -0,0 +1 @@
+*.o
```

Comment:
In such case, we also want to add `*.lo` (library object), `*.a` (static library archive) and `*.so` (shared library object).